src/metabase/driver.clj

```diff
@@ -885,7 +895,7 @@
 (defmethod syncable-schemas ::driver [_ _] #{})
 
 (defmulti upload-type->database-type
-  "Returns the database type for a given `metabase.upload` type."
+  "Returns the database type for a given `metabase.upload` type as a HoneySQL keyword."
```

Comment:
The return type is no longer just a HoneySQL keyword anymore because database-type can now be `[:varchar 255]` for mysql for example.